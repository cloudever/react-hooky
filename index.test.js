const rollup = require('rollup');
const pkg = require('./package.json');

let bundle;

it('contains all exports', async () => {
  bundle = await rollup.rollup({
    input: pkg.module,
    external: Object.keys(pkg.peerDependencies),
  });
});

it('match the latest shapshot', () => {
  expect(bundle.exports).toMatchSnapshot();
});
