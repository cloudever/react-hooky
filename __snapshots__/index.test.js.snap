// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match the latest shapshot 1`] = `
Array [
  "Audio",
  "Img",
  "Preload",
  "Script",
  "Stylesheet",
  "Video",
  "useDeviceMotion",
  "useDeviceOrientation",
  "useGeoPosition",
  "useIntersectionObserver",
  "useMedia",
  "useNetworkStatus",
  "useWindowScrollPosition",
  "useWindowSize",
  "useAsync",
  "useAudio",
  "useBattery",
  "useCounter",
  "useCss",
  "useFavicon",
  "useGeolocation",
  "useHover",
  "useIdle",
  "useLifecycles",
  "useList",
  "useLocalStorage",
  "useLocation",
  "useLogger",
  "useMap",
  "useMediaDevices",
  "useMotion",
  "useMount",
  "useNetwork",
  "useObservable",
  "useOrientation",
  "useRaf",
  "useSetState",
  "useSize",
  "useSpeech",
  "useSpring",
  "useTimeout",
  "useTitle",
  "useToggle",
  "useTween",
  "useUnmount",
  "useWindowSize2",
  "useMedia2",
  "useOnClickOutside",
]
`;
